cmd/start.go

```diff
@@ -198,6 +219,28 @@ func init() {
 		"20000:30000",
 		"[OPTIONAL] Range of ports which is used to open tunnels")
 
+	startCmd.Flags().StringVar(
+		&walletSecret,
+		"wallet-secret",
```

Comment:
@mpetrun5 @mpetrunic not sure if this is actually a good name for this?

Reply:
Does it expect mnemonic or private key?
Either way it think would be better to call it that way